src/modules/display.js

```diff
@@ -18,9 +19,14 @@ const filterLikes = (showId, likesList) => {
   }
   return result.likes;
 };
+
 const render = async () => {
   ul.innerHTML = '';
   const shows = await fetchShows();
+  const counterShows = () => shows.length;
```

Comment:
Kindly consider moving that function and exporting it above the render functions (say line 22) to make it more reusable and testable.